styles.css

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
If we're not going to use this commented code anymore, we could delete it in the next commit, what do you think?